package.json

```diff
@@ -28,9 +25,8 @@
   ],
   "license": "Apache-2.0",
   "scripts": {
-    "watch": "watchify src/main.js --standalone WebVRPolyfill -v -d -o 'derequire > build/webvr-polyfill.js'",
-    "build": "browserify src/main.js --standalone WebVRPolyfill | derequire > build/webvr-polyfill.js",
-    "min": "browserify src/main.js --standalone WebVRPolyfill | derequire | uglifyjs -c > build/webvr-polyfill.min.js",
+    "watch": "webpack --watch",
```

Comment:
also, if you decide to go with the changes I proposed above, make sure the `README.md` and `CONTRIBUTING.md` reference the correct script names